### DISCUSSION TITLE
Tickets Upload State/Save

### DISCUSSION THREAD

**Discussion by StingX2** [RetroAchievements/RAWeb#1570] [Ideas]
The only way to educate non devs to attach save states or save files which help devs fix things is to educate them. Their are many many non devs out there and they outnumber the devs tenfold. 

Like before when ticket manager was expanded to ask them about hashes we need to have a prompt asking for an upload of states/files. We have danced around a save state repository for dev fixing for years now but it starts here

If we caan implement this then this might be good to include too
"Are you sure you want to send this without a save state or save file demonstrating the issue? The problem may not get solved if you exclude these."

**Comment by Jamiras** [comment]
I think this is covered by #712
